uw-frame-components/portal/features/controllers.js

```diff
@@ -56,7 +56,7 @@ define(['angular','require'], function(angular, require) {
 
      var getPopups = function(){
        portalFeaturesService.getUnseenPopups().then(function(unSeenPopups) {
```

Comment:
I like how @paulerickson swoops in!  I'll take blame, as I did that originally... :(  I will also point out my inconsistency with the getUnseenPopups name and the unSeenPopups variable.